tests/usage.txt

```diff
@@ -1,25 +1,17 @@
-cargo-sweep.exe-sweep 
+Usage: cargo-sweep sweep [OPTIONS] <--stamp|--file|--time <TIME>|--installed|--toolchains <TOOLCHAINS>|--maxsize <MAXSIZE_MB>> [PATH]
 
-USAGE:
-    cargo-sweep.exe sweep [FLAGS] [OPTIONS] <--stamp|--file|--time <days>|--installed|--toolchains <toolchains>|--maxsize <maxsize>> [path]
+Arguments:
+  [PATH]  Path to check
 
-FLAGS:
-    -d, --dry-run      Dry run which will not delete any files
-    -f, --file         Load timestamp file in the given path, cleaning everything older
-    -h, --help         Prints help information
-        --hidden       The `recursive` flag defaults to ignoring directories that start with a `.`, `.git` for example
-                       is unlikely to include a Cargo project, this flag changes it to look in them.
-    -i, --installed    Keep only artifacts made by Toolchains currently installed by rustup
-    -r, --recursive    Apply on all projects below the given path
-    -s, --stamp        Store timestamp file at the given path, is used by file option
-    -V, --version      Prints version information
```

Comment:
Looks like `--version` has disappeared - that's unfortunate, it means `cargo sweep --version` no longer works. Is there a way to configure clap to allow it for the subcommand too?
```
; cargo sweep --version
error: unexpected argument '--version'

  note: argument '--verbose' exists

Usage: cargo-sweep sweep <--stamp|--file|--time <TIME>|--installed|--toolchains <TOOLCHAINS>|--maxsize <MAXSIZE_MB>> <PATH|--dry-run|--file|--hidden|--installed|--maxsize <MAXSIZE_MB>|--recursive|--stamp|--time <TIME>|--toolchains <TOOLCHAINS>|--verbose>
```

Reply:
Nice catch, I'll take a look and fix that.

Reply:
I added `#[command(about, version)]` attributes to the two clap access points, both `cargo-sweep` and the `sweep` subcommand, now it should work fine from the binary and when invoking from `cargo`.